MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+---
+layout: post
+title: Swiping in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Swiping support in Syncfusion .NET MAUI ListView (SfListView) control, its elements and more.
```

Comment:
description: Learn here all about swiping support in Syncfusion .NET MAUI ListView (SfListView) control, its elements, and more.

Reply:
resolved